src/AutoBuild.jl

```diff
@@ -1219,18 +1228,55 @@ function build_jll_package(src_name::String,
         using Pkg, Pkg.BinaryPlatforms, Pkg.Artifacts, Libdl
         import Base: UUID
 
+        wrapper_available = false
         \"\"\"
             is_available()
 
         Return whether the artifact is available for the current platform.
         \"\"\"
-        is_available() = false
+        is_available() = wrapper_available
```

Comment:
Maybe call the function `wrapper_available`?  I wasn't 100% happy with `is_available`

Reply:
I don't care about the name; I just care about the compiler warnings about precompilation being broken when we redefine a top-level method.  :P

If you want a different name, you can pick whatever you want. :)